ProcessMaker/Events/TemplateUpdated.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Carbon\Carbon;
+use Illuminate\Foundation\Events\Dispatchable;
+use ProcessMaker\Contracts\SecurityLogEventInterface;
+use ProcessMaker\Models\ProcessTemplates;
+use ProcessMaker\Traits\FormatSecurityLogChanges;
+
+class TemplateUpdated implements SecurityLogEventInterface
+{
+    use Dispatchable, FormatSecurityLogChanges;
+
+    private $changes;
```

Comment:
Define the type of variables  private $changes; private $process;  private $processType;